db/seeds.rb

```diff
@@ -1,7 +1,32 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+ # Create User
+user = User.new(
+    email:  "admin@example.com",
+    password:  "helloworld",
+    password_confirmation: "helloworld"
+  )
+user.skip_confirmation!
+user.save!
+
+# Create a list
+1.times do
+  @user = User.last
+  @user.lists.create!(
+    title: Faker::Lorem.sentence
+  )
+end
+
+#create items
+
+@list = List.last
```

Comment:
No need for @instance variable, a `list` local variable would be fine. Try to minimise variable scope where possible.